dl/src/main/scala/com/intel/analytics/bigdl/models/vgg/Train.scala

```diff
@@ -94,6 +94,10 @@ object Train {
         .setState(state)
         .setEndWhen(Trigger.maxEpoch(param.maxEpoch))
         .optimize()
+
+      if (sc.isDefined) {
```

Comment:
Looks like there are lots of example and we need a quick fix for user to have a try(see google user group). I have created a issue to track add similar code for other examples(See issue 565)